data.json

```diff
@@ -241,5 +241,14 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "dateAdded": "2018-10-12T13:05:05.000Z",
     "tags": ["stickers"]
+  },
+  {
+    "name": "7PHP",
```

Comment:
This file should be alphabetical sorted by name, please move this opportunity to the top ⬆